Orange/widgets/model/owunivariateregression.py

```diff
@@ -22,6 +22,9 @@ class OWUnivariateRegression(OWBaseLearner):
     name = "Univariate Polynomial Regression"
     description = "Univariate regression with polynomial expansion."
     icon = "icons/UnivariateRegression.svg"
+    replaces = [
+        "Orange.widgets.regression.owunivariateregression.OWUnivariateRegression",
+    ]
```

Comment:
This widget has been moved to educational add-on (and already improved there). Remove it from here. You can add `replaces` to the widget in the education add-on.